contribute/style-guides/documentation-style-guide.md

```diff
@@ -2,70 +2,69 @@
 
 ## Contributing
 
 The *Documentation style guide* is a living document. Add to it whenever a style decision is made or a question is answered regarding style, grammar, or word choice.
 
 ## Published guides
 
-For all items not covered in this guide, refer to the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/) and the [Chicago Manual of Style](https://www.chicagomanualofstyle.org/home.html).
+For all items that are not covered in this guide, refer to the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/) and the [Chicago Manual of Style](https://www.chicagomanualofstyle.org/home.html).
 
 ## Spelling
 
-The [codespell](https://github.com/codespell-project/codespell) tool is run for every change to catch common misspellings.
+To catch common misspellings, the [codespell](https://github.com/codespell-project/codespell) tool is run for every change.
 
 ## Inclusive language
 
-This section provides guidelines on how to avoid using charged language in documentation.
+Avoid using charged language.
 
 ### Allowing and blocking
 
-Don't use "whitelist" or "blacklist" when referring to allowing or blocking content or traffic.
+When referring to _allowing_ or _blocking_ content or traffic, use a form of _allow_ or _block_:
 
-- When used as a noun, use "allowlist" or "blocklist".
-- When used as a verb, use "allow" or "block"
+- (noun) _allowlist_ or _blocklist_
+- (verb) _allow_ or _block_
 
 Example: _To **allow** outgoing traffic, add the IP to the **allowlist**._
 
-### Leader and follower
-
-Don't use "master" or "slave" to describe relationships between nodes or processes.
+Avoid _whitelist_ or _blacklist_.
 
-- Use "leader", "main" or "primary," instead of "master."
-- Use "follower" or "secondary," instead of "slave."
+### Primary and secondary
 
-### Exceptions
+To describe relationships between nodes or processes, there are several options:
 
-When referring to a configuration or settings used by third-party libraries och technologies outside the Grafana project, prefer the original name to avoid confusion.
+- Use _primary_, _main_, or _parent_, instead of _master_.
+- Use _secondary_, _replica_, or _child_, instead of _slave_.
 
-For example, use "master" when referring to the default Git branch.
+Avoid _master_ or _slave_.
 
 ## Grafana-specific style
 
-The following sections provide general guidelines on topics specific to Grafana documentation. Note that for the most part, these are *guidelines*, not rigid rules. If you have questions, ask in the #docs channel of Grafana Slack.
+The following guidelines are specific to Grafana documentation. For the most part, these are *guidelines* are not rigid rules. If you have questions, ask in the #docs channel of Grafana Slack.
 
 ### General
 
-- Use active voice. Avoid passive voice.
+- Use active voice. Avoid passive voice:
```

Comment:
The period makes more sense here.

Reply:
Can you use the suggestion button so I can just click it. I agree with you. :)

Reply:
```suggestion
- Use active voice. Avoid passive voice.
```